apps/spruce/src/pages/task/taskTabs/logs/LogTypes.tsx

```diff
@@ -53,14 +54,19 @@ export const AllLog: React.FC<Props> = (props) => {
   const location = useLocation();
   const parsed = parseQueryString(location.search);
   const selectedExecution = Number(parsed[RequiredQueryParams.Execution]);
```

Comment:
nit. Not a requirement but it would be ⭐ _amazing_ ⭐ if we could replace these with `useQueryParam` while we are here.
```suggestion
  const [selectedExecution] = useQueryParam(RequiredQueryParams.Execution, 0);
```